.gitignore

```diff
@@ -8,3 +8,4 @@ gradle.properties
 *.iml
 *.iws
 out/
+test/
```

Comment:
We should fix the test that is generating something outside of the build directory instead of adding something to .gitignore